src/shared/creds-util.c

```diff
@@ -667,7 +667,7 @@ int encrypt_credential_and_warn(
         p += tsz;
         assert(p <= output_size);
 
-        if (DEBUG_LOGGING) {
+        if (DEBUG_LOGGING && input_size > 0) {
```

Comment:
Shouldn't this be `&& base64_size > input_size && input_size > 0` ?

Reply:
Indeed. Updated. PTAL.

Reply:
base64_size cannot really be smaller than input_size... we don't cut things of or compress hence result must be larger than input, given we add a header in front... 

or in other words, `base64_size >= input_size) sounds like something to test with an assert, not with an if check